pdfarranger/config.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (C) 2020 pdfarranger contributors
+#
+# pdfarranger is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+import platform
+import configparser
+import os
+from gi.repository import Gdk
+
+# See https://gitlab.gnome.org/GNOME/gtk/-/blob/3.24.23/gdk/keynames.txt for list of keys
+_DEFAULT_ACCELS=[
+    ('delete', 'Delete'),
+    ('page-format', 'c'),
+    ('rotate(90)', '<Ctrl>Right'),
+    ('rotate(-90)', '<Ctrl>Left'),
+    ('save', '<Ctrl>s'),
+    ('save-as', '<Ctrl><Shift>s'),
+    ('export-selection(2)', '<Ctrl>e'),
+    ('export-all', '<Ctrl><Shift>e'),
+    ('quit', '<Ctrl>q'),
+    ('import', '<Ctrl>o'),
+    ('zoom(5)', 'plus KP_Add'),
+    ('zoom(-5)', 'minus KP_Subtract'),
+    ('undo', '<Ctrl>z'),
+    ('redo', '<Ctrl>y'),
+    ('cut', '<Ctrl>x'),
+    ('copy', '<Ctrl>c'),
+    ('paste(0)', '<Ctrl>v'),
+    ('paste(1)', '<Ctrl><Shift>v'),
+    ('select(0)', '<Ctrl>a'),
+    ('select(1)', '<Ctrl><Shift>a'),
+    ('main-menu', 'F10'),
+]
+
+class Config(object):
+    """ Wrap a ConfigParser object for PDFArranger """
+
+    @staticmethod
+    def __get_action_list(m, r=[]):
```

Comment:
[Codacy actually spotted an error here](https://app.codacy.com/manual/jeromerobert/pdfarranger/file/49786423434/issues/source?bid=13283750&fileBranchId=20839044#l51): The empty list is only initialized once the class is loaded so for subsequent calls r already contains items from previous runs. Made this mistake once and it was a nightmare to figure it out. I think the best solution is `r = None` and `if not r: r = []`

Reply:
Thanks. Fixed. I just removed the default value.